orchagent/pfcwdorch.cpp

```diff
@@ -560,10 +560,15 @@ bool PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
                 sai_serialize_object_id(queueId));
     }
 
-    // Create egress ACL table group for each port of pfcwd's interest
-    sai_object_id_t groupId;
-    gPortsOrch->createBindAclTableGroup(port.m_port_id, groupId, ACL_STAGE_INGRESS);
-    gPortsOrch->createBindAclTableGroup(port.m_port_id, groupId, ACL_STAGE_EGRESS);
+    string platform = getenv("platform") ? getenv("platform") : "";
```

Comment:
Minor nitpick that someone pointed out in another PR, you can avoid doing `getenv` twice with something like this: https://github.com/Azure/sonic-swss/blob/d2bab10efe05e8ec209fd5a360778a2d7c8d7f54/orchagent/aclorch.cpp#L392

Reply:
Done